src/rustdoc-internals/rustdoc-gui-test-suite.md

```diff
@@ -7,6 +7,7 @@ For other rustdoc-specific test suites, see [Rustdoc test suites].
 
 These use a NodeJS-based tool called [`browser-UI-test`] that uses [puppeteer] to run tests in a headless browser and check rendering and interactivity. For information on how to write this form of test, see [`tests/rustdoc-gui/README.md`][rustdoc-gui-readme] as well as [the description of the `.goml` format][goml-script]
 
+
```

Comment:
nit: this linebreak doesn't need to be here

if its not too much trouble, i would appreciate you removing this extra linebreak then [squashing your commits](https://rustc-dev-guide.rust-lang.org/git.html#squash-your-commits) (after addressing all the other feedback) so the history for these files is a bit more clean (as is, this PR will show up as changing files it effectively doesn't)